components/AppMenu.vue

```diff
@@ -22,18 +27,18 @@ function getFlagEmoji(countryCode: string) {
     </nuxt-link>
     <div>
       <el-select
-        v-model="locale"
```

Comment:
`v-model` is highly reactive, thus triggers the locale change without the need of calling `setLocale`. We want more control, that's why we remove it.